src/picosvg/picosvg.py

```diff
@@ -50,5 +61,10 @@ def main():
     print(etree.tostring(tree, pretty_print=True).decode("utf-8"))
 
 
+def main():
+    # We don't seem to be __main__ when run as cli tool installed by setuptools
+    app.run(_run)
+
+
 if __name__ == "__main__":
-    main()
```

Comment:
this could stay simply `main()`, since the latter will call `app.run(_run)` anyway